README.md

```diff
@@ -12,8 +6,6 @@
 
 The Python scientific stack, compiled to WebAssembly.
 
-[**Try Pyodide and Iodide in your browser**](https://alpha.iodide.io/notebooks/300/)
-
```

Comment:
Removed as there is already a section on "Try Pyodide without installing" below and a link to iodide in the following section.